### PR TITLE
This time we only fixed errors, updated 8.2 on src/

### PR DESCRIPTION
**Pull request by caferyukseloglu**

### CODE REVIEW

**Comment by caferyukseloglu** [comment]
[Updated Php 8.2.1](https://github.com/c9s/Pux/pull/111/commits/99ddc6457274186f19d4ad1fd7da4648eb3b1e8a)
[Fixed issues](https://github.com/c9s/Pux/pull/111/commits/8b22810900e1258f73d761bd1014d4f881dca2cc)
This two is only desired changes

**Comment by caferyukseloglu** [comment]
@c9s

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ php:
 - 5.6
```
Comment (c9s) [inline .travis.yml:3]: travis is not working anymore, maybe we can just drop this

**Comment by c9s** [comment]
let's make this to version 3.0, we need to update the composer.json?

**Review by c9s — APPROVED** [review]

**Comment by caferyukseloglu** [comment]
Thank you and yes probably most of the travis and tests could be replaced

**Comment by caferyukseloglu** [comment]
> let's make this to version 3.0, we need to update the composer.json?

No changes in composer

**Pull request merged**